module/VuFind/tests/unit-tests/src/VuFindTest/ILS/Driver/PAIATest.php

```diff
@@ -56,6 +56,7 @@ class PAIATest extends \VuFindTest\Unit\ILSDriverTestCase
         'PAIA' =>
             [
                 'baseUrl'            => 'http://paia.gbv.de/',
+                'grantType'          => 'password',
```

Comment:
Should we also add some test cases to cover the new logic (config validation, etc.)?

Reply:
It should be fairly straightforward to mock the httpService and assert that appropriate headers are sent in when client credentials are configured... but it's definitely worth making sure @olli-gold agrees with the approach/implementation before spending time locking it in place with tests!